deployment/1.7/_index.md

```diff
@@ -6,6 +6,8 @@ summary: "First release for Jakarta EE 8"
 Jakarta Deployment defines standard APIs that will enable any deployment tool that uses the
 deployment APIs to deploy any assembled application onto a Jakarta EE compatible platform.
 
+This specification is a re-release of [JSR 88](http://jcp.org/en/jsr/detail?id=88) under the [EFSL](https://www.eclipse.org/legal/efsl/).
+
```

Comment:
Maybe OK, but just a comment.
Deployment 1.7 has not been released from JCP, nor included in Java EE 8.
Deployment 1.2 is the last version released from JCP.

Reply:
I'll specify that it is 1.2 which was in EE 8. Not sure why it suddenly jumped to 1.7 when it was moved...